frontend/src/components/Layout/AuthorizedLayout/Header/Header.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC } from 'react';
+
+import { PATHS } from 'constants/routes';
+import Avatar from 'components/Avatar';
+import Notifications from 'components/NotificationsBar';
+import { logOutIcon, menuMobileIcon } from 'icons';
+import { brandLogo } from 'images';
+import { Size } from 'enums/sizes';
+import { THeaderProps } from 'components/Layout/types';
+
+import {
+  LayoutHeader,
+  BrandLogoLink,
+  BrandLogo,
+  HeaderContent,
+  UserBlock,
+  UserName,
+  LogOut,
+  MobileMenuIcon,
+} from './styled';
+import SearchCourses from './SearchCourses';
+
+const Header: FC<THeaderProps> = ({
+  userInfo,
+  notifications,
+  isMobileMenuOpen,
+  handleConfirmLogOutOpen,
+  toggleMobileMenu,
+}) => {
+  const { avatar, firstName, lastName } = userInfo || {};
+
+  return (
+    <LayoutHeader container>
+      <BrandLogoLink to={PATHS.profile}>
+        <BrandLogo alt=":iTechArt" src={brandLogo} />
+      </BrandLogoLink>
+      <HeaderContent>
+        <SearchCourses />
+        <Notifications
+          notifications={notifications}
+          isMobileMenuOpen={isMobileMenuOpen}
+          toggleMobileMenu={toggleMobileMenu}
+        />
+        <UserBlock to={PATHS.profile}>
+          <Avatar avatar={avatar} size={Size.small} />
+          <UserName>{`${firstName} ${lastName}`}</UserName>
```

Comment:
maybe we should also implement a helper like on the BE side?
`export const convertToFullName = (firstName: string, lastName: string) => `${firstName} ${lastName}`;
`

Reply:
done